build.sbt

```diff
@@ -43,7 +43,7 @@ def wartSettings =
 
 lazy val commonSettings = Seq(
   organization := "org.scala-exercises",
-  version := "0.2.1-SNAPSHOT",
+  version := "0.2.2-SNAPSHOT",
```

Comment:
This PR will generate a new snapshot version. There are other PRs like https://github.com/scala-exercises/exercises-cats/pull/34 which are increasing the version. We have to do this for every module once 0.2.2 snapshot is released. In the meantime, you can use the 0.2.1.

Does it answer your question? Thaks for asking @fedefernandez